### PR TITLE
Refactor styling 

### PR DESCRIPTION
**Pull request by lrs8810**
- Fix styling on multiple show and index views
- All tests passing

### CODE REVIEW

File: app/views/admin/dashboard/index.html.erb [hunk 1]
```diff
@@ -3,7 +3,8 @@
 <% @orders.each do |order| %>
   <article id='order-<%= order.id %>'>
     <p><%= link_to "#{order.user.name}", "/admin/users/#{order.user.id}" %></p>
-    <p><%= "#{order.id}: #{order.status}" %></p>
+    <p><%= "Order ID: #{order.id}" %></p>
+    <p><%= "Order Status: #{order.status}"%></p>
```
Comment (CoopTang) [inline app/views/admin/dashboard/index.html.erb:7]: I like that this is more explicit.

Comment (freeheeling) [inline app/views/admin/dashboard/index.html.erb:7]: Improved presentation.

File: app/views/merchants/index.html.erb [hunk 2]
```diff
@@ -9,12 +9,12 @@
...
           <% if merchant.enabled? %>
             <%= link_to 'Disable', admin_merchants_path(merchant), method: :patch %>
-          <% else %> 
+          <% else %>
             <%= link_to 'Enable', admin_merchants_path(merchant), method: :patch %>
           <% end %>
         </p>
       <% else %>
-        <p><%= link_to merchant.name, merchants_path(merchant) %></p>
+        <p><%= link_to merchant.name, merchant_path(merchant) %></p>
```
Comment (CoopTang) [inline app/views/merchants/index.html.erb:17]: Good catch! We wouldn't have found this without poking around the site.

**Review by freeheeling — COMMENTED** [review]
Approved.

**Pull request merged**